.github/workflows/gatk-tests.yml

```diff
@@ -37,17 +50,27 @@ jobs:
           tags: |
             type=raw,value=${{github.sha}}
 
+      - name: Set up Docker Buildx
```

Comment:
Xtra x

Reply:
Actually no. Docker buildx is what they call their special build script.

Reply:
Xactly the Xorrect number of x's then